ansible/olm_cleanup.yaml

```diff
@@ -2,18 +2,26 @@
 - hosts: convergence_base
   become: true
   become_user: ocp
+  vars_files: "vars/default.yaml"
+  environment:
+    PATH: "/usr/local/bin:{{ ansible_env.PATH }}"
+    KUBECONFIG: "{{base_path}}/dev-scripts/ocp/{{ ocp_cluster_name }}/auth/kubeconfig"
 
   tasks:
   - name: Include variables
     include_vars: vars/default.yaml
 
+  - name: switch to openstack project/namespace
+    ignore_errors: true
+    shell: >
+      oc project openstack
```

Comment:
we spoke about getting rid of these project changes entirely and instead just scoping each oc command with the proper namespace. oc delete pod -n openstack for example

Reply:
👍

Reply:
for now, this matches what is in olm.yaml. we can change it up in a different patch.